curriculum/challenges/english/25-front-end-development/lecture-understanding-functional-programming/6733b0451d6be0065430b418.md

```diff
@@ -2,13 +2,15 @@
 id: 6733b0451d6be0065430b418
 title: What Is Functional Programming?
 challengeType: 19
-# videoId: nVAaxZ34khk
+videoId: c7fPDnn80Co
```

Comment:
while reviewing the video found a couple of graphic errors. Waiting for the new video and then I will replace the video id